lnwire/error.go

```diff
@@ -15,24 +15,17 @@ const (
 	// active pending channels exceeds their maximum policy limit.
 	ErrMaxPendingChannels FundingError = 1
 
-	// ErrSynchronizingChain is returned by a remote peer that receives a
-	// channel update or a funding request while it's still syncing to the
-	// latest state of the blockchain.
-	ErrSynchronizingChain FundingError = 2
-
 	// ErrChanTooLarge is returned by a remote peer that receives a
 	// FundingOpen request for a channel that is above their current
 	// soft-limit.
-	ErrChanTooLarge FundingError = 3
+	ErrChanTooLarge FundingError = 2
```

Comment:
@Crypt-iQ so even though this is declared in the `lnwire` package, this constant is never used on the wire? So changing the number shouldn't have any side effects? I guess we only created the type for internal handling then...

Reply:
yes its not used on the wire